src/index.js

```diff
@@ -24,3 +24,18 @@ export function doubleEach ( arr ) {
   return result;
 }
 
+export function Animal() {}
+Animal.prototype.speak = 'generic sound';
+
+Reptile.prototype = Object.create(Animal.prototype);
+export function Reptile() {}
+Reptile.prototype.speak = () => console.log('Hisss!');
```

Comment:
It doesn't affect these tests, but for technical reasons, you don't want to use a fat arrow here. See if you can figure out why before class (then ask me about it in anyway). As a hint, the fat arrow is formally known as the "function bind" or "lexical `this`" syntax.